src/Effects.js

```diff
@@ -1018,198 +1149,226 @@ function drawFallingPoop(ctx) {
   ctx.save();
   ctx.fillStyle = "#995243";
   ctx.beginPath();
-  ctx.moveTo(3.378,11.244);
-  ctx.lineTo(14.622,11.244);
-  ctx.quadraticCurveTo(18,11.244,18,14.622);
-  ctx.lineTo(18,14.620999999999999);
-  ctx.quadraticCurveTo(18,17.999,14.622,17.999);
-  ctx.lineTo(3.378,17.999);
-  ctx.quadraticCurveTo(0,17.999,0,14.620999999999999);
-  ctx.lineTo(0,14.622);
-  ctx.quadraticCurveTo(0,11.244,3.378,11.244);
-  ctx.closePath();
-  ctx.fill();
-  ctx.stroke();
-  ctx.restore();
-  ctx.save();
-  ctx.fillStyle = "#995243";
-  ctx.transform(1,0,-0.00017,1,2.104,5.87);
-  ctx.beginPath();
-  ctx.moveTo(3.592,0);
-  ctx.lineTo(10.179,0);
-  ctx.quadraticCurveTo(13.771,0,13.771,3.592);
-  ctx.lineTo(13.771,3.592);
-  ctx.quadraticCurveTo(13.771,7.184,10.179,7.184);
-  ctx.lineTo(3.592,7.184);
-  ctx.quadraticCurveTo(0,7.184,0,3.592);
-  ctx.lineTo(0,3.592);
-  ctx.quadraticCurveTo(0,0,3.592,0);
-  ctx.closePath();
-  ctx.fill();
-  ctx.stroke();
-  ctx.restore();
-  ctx.save();
-  ctx.fillStyle = "#995243";
-  ctx.beginPath();
-  ctx.moveTo(9.21,2.183);
-  ctx.bezierCurveTo(6.843,2.603,5.45,0,5.45,0);
-  ctx.bezierCurveTo(4.062,2.411,4.166,5.546,6.356,7.264);
-  ctx.bezierCurveTo(7.734,8.345,9.317,9.11,10.58,8.406);
-  ctx.translate(10.29000831352916,5.243266874714561);
+  ctx.moveTo(5.449,0);
```

Comment:
What is the source of all the SVG code in this file?

Reply:
mark made these SVG drawings for us.

Reply:
Can we document the pipeline used here so that future maintainers know how to work with it?

I was worried this was hand-coded :)

It's also good to know whether we can round-trip the assets (from code back into tool), but it looks like it's one-directional here, which means knowing where they came from is quite important.